microsoft-edge/toc.yml

```diff
@@ -1029,89 +1046,167 @@
+                  href: extensions-chromium/developer-guide/manifest-v3.md
+
+                - name: Migrate an extension from Manifest V2 to V3
+                  href: extensions-chromium/developer-guide/migrate-your-extension-from-manifest-v2-to-v3.md
+
+                - name: Manifest file format for extensions
+                  href: extensions-chromium/getting-started/manifest-format.md
+
+            - name: JavaScript APIs
+              items:
+                - name: Microsoft Edge Add-ons API 101
+                  href: extensions-chromium/developer-guide/apis-intro.md
+
+                - name: Supported APIs for Microsoft Edge extensions
+                  href: extensions-chromium/developer-guide/api-support.md
+
+                - name: Declare API permissions in extension manifests
+                  href: extensions-chromium/developer-guide/declare-permissions.md
 
-            - name: Migrate an extension from Manifest V2 to V3
-              href: extensions-chromium/developer-guide/migrate-your-extension-from-manifest-v2-to-v3.md
+            - name: Policies for extensions
+              items:
+                - name: Content Security Policy (CSP)
+                  href: extensions-chromium/store-policies/csp.md
 
-            - name: Manifest file format for extensions
-              href: extensions-chromium/getting-started/manifest-format.md
+                - name: App Developer Agreement Addendum for Microsoft Edge program users
+                  href: extensions-chromium/store-policies/ada-addendum.md
 
-        - name: Publish to the Add-ons website
+                - name: Developer policies for the Microsoft Edge Add-ons store
+                  href: extensions-chromium/store-policies/developer-policies.md
+
+# -------------------------------------
+        - name: Publish an extension
           items:
-            - name: Register as a Microsoft Edge extension developer
-              href: extensions-chromium/publish/create-dev-account.md
+            - name: Register as an individual developer
+              items:
+                - name: Register as a Microsoft Edge extension developer
+                  href: extensions-chromium/publish/create-dev-account.md
+
+            - name: Register as an enterprise developer
+              items:
+#               - name: Register as a Microsoft Edge extension developer for Company Account
+#                 href:
 
-            - name: Verify your company account information
-              href: extensions-chromium/publish/verify-microsoft-edge-program.md
+                - name: Verify your company account information
+                  href: extensions-chromium/publish/verify-microsoft-edge-program.md
+                  displayName: Verify the company account information associated with your Microsoft Partner Center account  # todo: shorten top-of-article title, clarify it in first sentence instead
```

Comment:
disparity: toc title:
Verify your company account information
top-of-article title:
Verify the company account information associated with your Microsoft Partner Center account

proposal: shorten top-of-article title (in all the usual spots).  make sure first sentence in article clarifies title.

Reply:
globally in all the usual spots, changed title to:
Verify your company account information
1st sentence in article is ok, to clarify title.